.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules
 npm-debug.log
 yarn-error.log
 coverage
+typings
```

Comment:
We don't use typings anymore, instead can you remove typings folder in your local